main.go

```diff
@@ -105,6 +106,11 @@ func getCliApp() *cli.App {
 			Name:  "subscribe",
 			Usage: "Subscribe to Rancher events",
 		},
+		cli.Int64Flag{
+			Name:  "reload-interval-limit",
+			Usage: "Limits reload to 1 per interval (milliseconds)",
+			Value: 100,
```

Comment:
I would assume 100 milliseconds won't make much of a different, what about 1000-2000?

Reply:
I just thought of using this default to be close to the old behavior, but sure, will change it to 1 sec.